workLoggingSystem.html

```diff
@@ -235,7 +235,13 @@
                                 keyboard_backspace
                             </span>
                         </div>
-                        <div class="col"></div>
+                        <div class="col">
+                            <div class="row">
+                            <div class="col-md-1">Date: </div>
+                            <div class="col" id="showDate">
```

Comment:
better to call creationDateCreateNewTask rather than showDate and make the changes accordingly

Reply:
okay, i will update with the changes.